src/psyclone/tests/psyad/domain/lfric/test_lfric_adjoint_harness.py

```diff
@@ -67,6 +68,12 @@ def lfric_consts_fixture():
     return LFRicConstants().DATA_TYPE_MAP
 
 
+@pytest.fixture(scope="module", name="lfric_config", autouse=True)
+def fixture_lfric_config():
```

Comment:
Moved the fixture. But not added the clear up comment since I have been deleting it from everywhere else, as it is the new normal/expected, unless you disagree.